source/form.html

```diff
@@ -0,0 +1,201 @@
+
+      <h1 class="form-container__title">Подбор программы</h1>
+      <p class="form-container__text">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+
+      <form class="form-container__form order-form" action="#" method="post">
+
+        <fieldset class="order-form__group form-group">
+          <legend class="visually-hidden">Данные кота</legend>
+
+          <p class="form-group__item">
+            <label class="form-group__field field">
+              <span class="field__text">Имя:*</span>
+              <input class="field__input" name="name" type="text" placeholder="Барсик">
+            </label>
+          </p>
+
+          <p class="form-group__item">
+            <label class="form-group__field field">
+              <span class="field__text">Вес(кг):*</span>
+              <input class="field__input" name="weight" type="text" placeholder="7">
+            </label>
+          </p>
+
+          <p class="form-group__item">
+            <label class="form-group__field field">
+              <span class="field__text">Возраст (лет):*</span>
+              <input class="field__input" name="age" type="text" placeholder="7">
+            </label>
+          </p>
+
+        </fieldset>
+
+        <fieldset class="order-form__group form-group">
+          <legend class="visually-hidden">Выбор программы питания</legend>
+
+          <ul class="form-group__list">
+
+            <li class="form-group__item">
+              <label class="form-group__field radio">
+                <input class="radio__input" name="programm" type="radio" value="diet" checked>
+                <span class="radio__text">Похудение</span>
+              </label>
+            </li>
+
+            <li class="form-group__item">
+              <label class="form-group__field radio">
+                <input class="radio__input" name="programm" type="radio" value="weight-gain">
+                <span class="radio__text">Набор массы</span>
+              </label>
+            </li>
+
+            <li class="form-group__item">
+              <label class="form-group__field radio">
+                <input class="radio__input" name="programm" type="radio" value="advice">
+                <span class="radio__text">Не знаю (Нужен ваш совет)</span>
+              </label>
+            </li>
+
+          </ul>
+
+        </fieldset>
+
+        <fieldset class="order-form__group form-group">
+          <legend class="form-group__title">Контактные данные (владельца кота)</legend>
+
+          <p class="form-group__item">
+            <label class="form-group__field field">e-mail:*
+              <input class="field__input" name="email" type="email" placeholder="kuklachev@gmail.com">
+            </label>
+          </p>
+
+          <p class="form-group__item">
+            <label class="form-group__field field">Телефон:*
+              <input class="field__input" name="phone" type="tel" placeholder="8 (960) 900-60-90">
+            </label>
```

Comment:
Очень не рекомендую ставить контент внутри тега в строку с открывающей частью, если внутри уже на отдельных строках все размещено:
```suggestion
            <label class="form-group__field field">
              Телефон:*
              <input class="field__input" name="phone" type="tel" placeholder="8 (960) 900-60-90">
            </label>
```

Большая вероятность того, что такой текст в конце тега останется незамеченным